lib/propolis/src/enlightenment/mod.rs

```diff
@@ -0,0 +1,95 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! Components that implement enlightenments: mechanisms that allow guest
```

Comment:
suuuuuper nitpicky, feel free to totally disregard me: since we are introducing the term "enlightment" here, i might italicize it?
```suggestion
//! Components that implement _enlightenments_: mechanisms that allow guest
```

Reply:
Fixed in c185b4b.